cmake/compilers/Clang.cmake

```diff
@@ -48,5 +48,7 @@ if (ANDROID_PLATFORM)
     set(TBB_COMMON_COMPILE_FLAGS ${TBB_COMMON_COMPILE_FLAGS} $<$<NOT:$<CONFIG:Debug>>:-D_FORTIFY_SOURCE=2>)
 endif()
 
+set(TBB_IPO_LINK_FLAG -flto)
```

Comment:
Doesn't `Clang` require `-flto` flag during the compile stage too?

Reply:
Yes, it does, thanks!